src/helperFunctions/uid.py

```diff
@@ -1,46 +1,47 @@
 import re
+from typing import AnyStr, List, Set, Union
 
 from helperFunctions.dataConversion import make_bytes
 from helperFunctions.hash import get_sha256
 
 
-def create_uid(input_data):
+def create_uid(input_data: AnyStr) -> str:
     '''
-    creates an unique identifier: SHA256_SIZE
+    generate a UID (unique identifier) SHA256_SIZE for a string containing data
```

Comment:
Hm. I thought we'd encounter bytes here not strings. What would the sense be to generate this based on a string?

Reply:
this is actually used e.g. in `db_interface_frontend_editing.py` with a string but changing that is probably out of scope. Changed the doc string